lib/moyasar/actions/refund.rb

```diff
@@ -0,0 +1,23 @@
+module Moyasar
+  module Actions
+    module Refund
+
+      def self.included(klass)
+        klass.extend         ClassMethods
+        klass.send :include, InstanceMethods
+      end
+
+      module ClassMethods
+        def refund(id, attrs = {})
+          response = request(:post, refund_url(id), params: attrs)
+
+          new(response.body)
+        end
+      end
+
+      module InstanceMethods
```

Comment:
Perfect! I liked how you done it. Do we need separate test for instance refund method or what we have was enough?